### ISSUE TITLE
UI Enhancement for a More Professional and Premium Design

### ISSUE THREAD

**Issue by TheOpenInnovator** [TheOpenInnovator/ChromaVerse#2]
**Description**:  
The current UI of the ChromaVerse website, while functional, needs refinement to appear more professional and premium. To elevate the user experience, consider the following improvements:

1. **Typography**: Update the font to a more modern and clean typeface. Consider using Google Fonts like "Poppins" or "Roboto" for a more polished look.
2. **Color Palette**: Refine the color scheme for better contrast and consistency. Ensure the primary and secondary colors complement each other to create a cohesive theme.
3. **Spacing and Layout**: Add more whitespace to improve readability and balance. Refine the layout to make it more responsive and centered, ensuring a sleek look on all devices.
4. **Buttons and Inputs**: Update buttons with rounded corners, subtle shadows, and hover animations. Make the input fields more prominent and stylish.
5. **Glassmorphism and Shadows**: Apply advanced glassmorphism techniques and soft shadows for depth, enhancing the modern and professional feel.
6. **Animations and Transitions**: Introduce smooth transitions for hover states, palette generation, and modal pop-ups, providing a fluid and interactive experience.
7. **Dark Mode Improvements**: Ensure the dark mode looks polished and seamless, with proper contrast between text and background.

---

**Comment by swarupn17** [comment]
Hey @TheOpenInnovator I can build better UI for the website. Please can you assign it to me.
Thanks !

**Comment by TheOpenInnovator** [comment]
Hey @swarupn17 Any updates on the Issue??

**Comment by harith-hacky03** [comment]
I'm ready to fix it asap . Can you assign me ?

**Comment by TheOpenInnovator** [comment]
Sure @harith-hacky03 You can work on this issue. Do prefer the instructions mentioned. The issue has been assigned to you.

**Comment by harith-hacky03** [comment]
Yeah i created the PR let me know if any changes is needed thank you!

**Comment by deepak-raaaz** [comment]
@TheOpenInnovator Hi! I've reviewed your application on my system, and I'm pleased to share that I can help redesign the UI and add some extra sections to enhance its functionality and appeal. Let me know how you'd like to proceed!

**Comment by TheOpenInnovator** [comment]
> @TheOpenInnovator Hi! I've reviewed your application on my system, and I'm pleased to share that I can help redesign the UI and add some extra sections to enhance its functionality and appeal. Let me know how you'd like to proceed!

Sure @deepak-raaaz You can create a new PR based on the requirements & start working.

**Comment by lavbingi** [comment]
is this done or still to be done assign me if not done
